tests/unit_tests/config/test_queue_config.py

```diff
@@ -185,3 +184,23 @@ def test_undefined_LSF_SERVER_environment_variable():
         ),
     ):
         ErtConfig.from_file(filename)
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+@pytest.mark.parametrize(
+    "queue_system, queue_system_option",
+    [("LSF", "LSF_SERVER"), ("SLURM", "SQUEUE"), ("TORQUE", "QUEUE")],
+)
+def test_initializing_empty_config_values(queue_system, queue_system_option):
+    filename = "config.ert"
+    with open(filename, "w", encoding="utf-8") as f:
+        f.write("NUM_REALIZATIONS 1\n")
+        f.write(f"QUEUE_SYSTEM {queue_system}\n")
+        f.write(f"QUEUE_OPTION {queue_system} {queue_system_option}\n")
+        f.write(f"QUEUE_OPTION {queue_system} MAX_RUNNING\n")
+    config_object = ErtConfig.from_file(filename)
+    driver = Driver.create_driver(config_object.queue_config)
+    assert driver.get_option(queue_system_option) == ""
+    assert driver.get_option("MAX_RUNNING") == "0"
+    for options in config_object.queue_config.queue_options[queue_system]:
+        assert isinstance(options, tuple)
```

Comment:
Is this leftovers, or can options be something else than a `Tuple[str,str]` ?

Reply:
Just there to verify that every option is now a tuple when giving in empty values.